src/Config/EventConfigurator.php

```diff
@@ -59,6 +80,17 @@
             return true;
         }
 
-        return is_array($definition) && array_keys($definition) === [0, 1] && in_array($definition[1], get_class_methods($definition[0]) ?? [], true);
+        if (
+            is_array($definition)
+            && array_keys($definition) === [0, 1]
+            && is_string($definition[0])
+            && $this->container->has($definition[0])
+        ) {
+            $object = $this->container->get($definition[0]);
```

Comment:
I think this method should not throw an exception

Reply:
This method does not throw an exception outside its object. These exceptions are catched above and chained into a more suitable exception.